spec/requests/swagger_docs/merits_tasks_spec.rb

```diff
@@ -3,6 +3,21 @@
 RSpec.describe "merits_tasks", type: :request, swagger: true do
   path "/merits_tasks" do
     post("Return details of merits_tasks for specified proceeding types identified by CCMS-codes") do
+      description "POST a JSON payload containing request-id (a UUID generated by the client) and an array
+                   of proceeding types (identified by ccms_code) to
+                   return an response containing the same request-id and an array of merits taks, grouped
+                   by application and proceeding type. <br/><br/>
+                   The tasks listed under application must be done once for the entire application, the tasks listed under
+                   each proceeding type must be done once for each proceeding type.  Each task is specified as a key-value
+                   pair where the key is the taks name, and the value is the tasks on which it is dependent, or an empty array
```

Comment:
Typo : `where the key is the taks name,`
```suggestion
                   pair where the key is the task name, and the value is the tasks on which it is dependent, or an empty array
```